QueryLib.js

```diff
@@ -0,0 +1,60 @@
+define([], function() {
+  /////////////////////////////////////////////////////////////////////////////
+  //
+  // Private members
+  //
+  /////////////////////////////////////////////////////////////////////////////
+
+  // Set up statistics definition for client-side query
+  // Total popultion of age groups by gender in census tracts
+  const statDefinitions = ["Hosp_beds", "Occupency"].map(function(fieldName) {
+    return {
+      onStatisticField: fieldName,
+      outStatisticFieldName: fieldName + "_TOTAL",
+      statisticType: "sum"
+    };
+  });
+
+  /////////////////////////////////////////////////////////////////////////////
+  //
+  // Public (exported) members
+  //
+  /////////////////////////////////////////////////////////////////////////////
+
+  return {
+    //Spatial query for hospital feature layer view for statistics
+    queryLayerViewAgeStats(featureLayerView, buffer) {
```

Comment:
This is an exported method; note how it is defined on an object that is `return`ed from the AMD module.